tests/jobs/localisation/example_case/scripts/init_test_case.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+"""
+Script initialize the test case by creating the grid files, observation files etc
+"""
+
+import math
+import os
+import random
+
+# pylint: disable=import-error
+import xtgeo
+from common_functions import generate_field_and_upscale, settings, write_upscaled_field
+
+# pylint: disable=too-many-arguments,invalid-name,missing-function-docstring
+# pylint: disable=too-many-locals,redefined-outer-name
+
+
+def generate_seed_file(
+    start_seed: int = 9828862224,
+    number_of_seeds: int = 1000,
+):
+    # pylint: disable=unused-variable
```

Comment:
Instead of disabling pylint here, we can rewrite the for-loop as follows:

`for _ in range(number_of_seeds):`.

The underscore indicates a variable that's not meant to be used.

Reply:
fixed